Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,10 @@
-import React from 'react';
-import './Button.css';
+import React from "react";
+import "./Button.css";
+const ActionButton = props => {
+  return (
+    <div className={`${props.actionProps.style}`}>
```

Comment:
Something to think about: does making className dynamic here makes the code cleaner/easier to understand than a string?